.github/workflows/github-pages.yml

```diff
@@ -8,11 +8,11 @@ jobs:
     runs-on: ubuntu-20.04
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
     - name: Set up Python
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v5
       with:
-        python-version: '3.6'
+        python-version: '3.12'
```

Comment:
I have a feeling the sphinx-versioning thing won't work under python 3.12. Have you tested it locally?

I still have to fix the whole versioning thing and do it differently, same for mpire. But didn't take the time yet to fix that. If it's not working locally, it's best to revert to 3.6 for the docs for now

Reply:
The documentation did build on 3.12 without errors. But I can revert back to 3.6 if you like. Or shall I do 3.8 as that's now the lowest version TextScrubber supports?